src/user-guide/interventions/index.rst

```diff
@@ -488,3 +488,37 @@ Hold The Workflow And Trigger Tasks One By One
    Task Hold
       When a task is :term:`held <hold>`, then it will not submit (if ready to
       submit) until released.
+
+
+I want to Skip a cycle of tasks and allow the workflow to continue
+------------------------------------------------------------------
+
+:Example:
+
+   I want to skip a cycle (or group) of tasks and continue as if they had run
+   and succeeded.
+
+:Solution:
+
+   Set the run mode of the tasks to skip and Cylc will pretend that they
+   have run.
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: skip-cycle.gui.gif
+         :width: 75%
+
+   .. tab-item:: CLI
+      :sync: cli
+
+      .. code-block:: console
+
+         cylc broadcast -p '<cycle-glob>' -n root -s 'run mode = skip'
```

Comment:
Cylc broadcast doesn't actually accept globs. It does except the `*` wildcard, but you can't actually glob with it (e.g. `2000*`).

Maybe add to the note explaining that `root` matches all tasks, but that this could be swapped out for more targeted interventions.